index.js

```diff
@@ -1605,10 +1609,10 @@
                      (t.url)
                      (t.supertypes)
                      (t._test ([]))
-                     (t.extractors.$1);
+                     (t._extractors.$1);
```

Comment:
We provide the user-provided extractor function here to avoid `B (toArray) (B (toArray) (_1))`.